highway_env/envs/common/graphics.py

```diff
@@ -94,12 +94,12 @@ def handle_events(self) -> None:
             if self.env.action_type:
                 EventHandler.handle_event(self.env.action_type, event)
 
-    def display(self) -> None:
+    def display(self, center=True) -> None:
         """Display the road and vehicles on a pygame window."""
         if not self.enabled:
             return
 
-        self.sim_surface.move_display_window_to(self.window_position())
+        self.sim_surface.move_display_window_to(self.window_position() if center else [0.5, 0.5])
```

Comment:
I don't think you should edit these common files in abstract and graphics, instead just subclass the window_position() method in the environment to customise it.
Also, I don't understand why you use [0.5, 0.5] as a centre position, this is in meters so it looks like a strange choice to me.